backtrace-library/src/main/java/backtraceio/library/models/BacktraceData.java

```diff
@@ -98,103 +97,198 @@ public class BacktraceData {
     /**
      * Current BacktraceReport
      */
-    public transient BacktraceReport report;
-
-    /**
-     * Current application context
-     */
-    public transient Context context;
+    public transient BacktraceReport report; // Think if we need it
 
     /**
      * Application thread details
      */
     @SerializedName("threads")
     Map<String, ThreadInformation> threadInformationMap;
+    
+    public BacktraceData() { }
 
-    /**
-     * Create instance of report data
-     *
-     * @param context          current application context
-     * @param report           current report
-     * @param clientAttributes attributes which should be added to BacktraceData object
-     */
-    public BacktraceData(Context context, BacktraceReport report, Map<String, Object>
```

Comment:
do we need to keep the previous constructor to make sure we don't introduce any breaking change?

Reply:
I would say let's drop this constructor because we will not be able to set proper values -> I've moved all logic of preparing values to Builder class, let's put it clearly in CHANGELOG.md that it will not be backward compatible.